.forge-snapshots/swap with return dynamic fee.snap

```diff
@@ -0,0 +1 @@
+157979
```

Comment:
2,261 gas less than the previous implementation (no return fee, and have beforeSwap call `updateDynamicLPFee`)

Reply:
Im seeing 2,818 difference.. comparing this number to the snapshot below

Reply:
no i agree with sauce - you have to take the _old_ snapshot of the snapshot below, not the new one

Reply:
oh sorry yes that makes sense haha